huxley/www/static/js/huxley/components/AdvisorRosterView.js

```diff
@@ -109,6 +114,51 @@ var AdvisorRosterView = React.createClass({
     }.bind(this));
   },
 
+  _addDelegatePressed: function() {
+    return (
+      <form>
+        Name: <input type="text" placeholder="Name" valueLink={this.linkState('name')} /><br>
+        Email: <input type="text" placeholder="Email" valueLink={this.linkState('email')}/><br>
+        Summary: <input type="text" placeholder="Summary" valueLink={this.linkState('summary')}/><br>
```

Comment:
No need for inputting a summary. This is a feature advisors don't have control over. Remove this line